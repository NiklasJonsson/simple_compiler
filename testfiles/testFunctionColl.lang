int main() {
	fun1();
	fun2();
}

int fun(){ fun2();}
int fun2(){ fun1();}
int fun1(){ int hje = 0;}