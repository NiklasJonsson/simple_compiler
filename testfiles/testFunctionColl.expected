main:
	fun1
	fun2
fun:
	fun2
fun2:
	fun1
fun1:
